test/acceptance/framework/logger/logger.go

```diff
@@ -0,0 +1,43 @@
+package logger
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	terratestTesting "github.com/gruntwork-io/terratest/modules/testing"
+)
+
+// TestLogger implements terratest's TestLogger interface
+// so that we can pass it to terratest objects to have consistent logging
+// across all tests.
```

Comment:
This is awesome!! I'm going to try to summarize my understanding of how to use this logger for future acceptance test code, and please do correct me if I'm wrong. In our own test helper methods, we should use `logger.Logf(...)`. In any terratest methods we leverage, we would pass it a logger `terratestLogger.New(logger.TestLogger{})` if that method in terratest already accepts a logger. Or if the method in terratest does not accept a logger, we'd reimplement it to take a logger, as described in the comment on line 12 in `test/acceptance/framework/k8s/kubectl.go`.

Reply:
Yup exactly!